packages/discovery-provider/src/api/v1/users.py

```diff
@@ -3117,8 +3117,25 @@ def get(self, id, authed_user_id):
         return success_response(muted_users)
 
 
-email_key_response = make_response(
-    "email_key_response", ns, fields.String(required=False, allow_null=True)
+email_access_response = make_response(
+    "email_access_response",
+    ns,
+    fields.Nested(
+        ns.model(
```

Comment:
nit: do we want to put this in `packages/discovery-provider/src/api/v1/models/users.py`?